Headers/cairo/WaylandCairoSurface.h

```diff
@@ -0,0 +1,16 @@
+/*
+   Copyright (C) 2016 Sergio L. Pascual <slp@sinrega.org>
```

Comment:
Could we change all the copyright messages here and add the standard GNUstep header?

Reply:
Done. We'll need a review from @slp -- I think copyright assignment happened so that the copyright ownership transfer was intended, but let's confirm this.

Reply:
@ivucica Sure! Please change that to the standard GNUstep header.